tools/githooks/lib/src/pre_push_command.dart

```diff
@@ -81,6 +77,22 @@ class PrePushCommand extends Command<bool> {
     return true;
   }
 
+  io.File? _findCompileCommands(String flutterRoot) {
+    for (final String dir in _checkForHostTargets) {
```

Comment:
Maybe choose the one with the most recent file modification time? (Not sure if modification or creation time would be better...) Which one was chosen should probably also be printed to the terminal.